examples/cuda/image_function/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(cuda_example_image_function
```

Comment:
We don't need to commit this file as well as `examples/cuda/pinned_memory/CMakeLists.txt`. They are invalid for CUDA compilation so no need to commit them at all. Please remove them from the pull request.